src/features/island/buildings/components/building/workBench/components/ToolsGuide.tsx

```
@@ -269,17 +294,23 @@ const NodeRow: React.FC<NodeRowProps> = ({
     game: state,
   });
 
+  const resourceName =
+  ITEM_DETAILS[node.resourceName]?.translatedName ?? node.resourceName;
+
```

ESLint

[error] Replace `ITEM_DETAILS[node.resourceName]?.translatedName·??·node.resourceName;⏎` with `··ITEM_DETAILS[node.resourceName]?.translatedName·??·node.resourceName;`

(prettier/prettier)